project/frontend/src/components/PreferenceForm.js

```diff
@@ -71,6 +71,12 @@ function PreferenceForm(props) {
     })();
   }, [currLocation]);
 
+  useLayoutEffect(() => {
+    setTooltipVal(radiusWeight, 0);
+    setTooltipVal(diningExpWeight, 1);
+    setTooltipVal(priceLevelWeight, 2);
+  }, [radiusWeight, diningExpWeight, priceLevelWeight]);
+
```

Comment:
I feel like doing something with a loop would be better for future-proofing this code. Idk if we'll actually get around to it, but if we were to add more preferences, this method of doing it could get really verbose.

Event simply passing an array to `setTooltipVal` I think would be a lot better. For example:
```
setTooltipVal([radiusWeight, diningExpWeight, priceLevelWeight]);
```
and then inside `setTooltipVal`:
```
for (const tooltip of tooltips) {
  tooltips[tooltipInd].innerHTML = levels[val - 1];
}
```

Reply:
This is a great idea! Done in latest commit (a90ad05)